src/etc/inc/util.inc

```diff
@@ -1004,6 +1004,28 @@ function alias_expand($name) {
 	}
 }
 
+/* get description of alias */
+function get_alias_description($name)
+{
+    global $config;
+
+    if (is_alias($name))
+    {
```

Comment:
it should be save to ditch is_alias, the block below does approx. the same thing :)